rmf_fleet_adapter/schemas/event_description__charge.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://raw.githubusercontent.com/open-rmf/rmf_ros2/main/rmf_fleet_adapter/schemas/event_description__charge.json",
+  "title": "Charge Event",
```

Comment:
I recommend we add "Battery" everywhere to make this more explicit, e.g. "Charge Battery Event", `event_description__charge_battery.json`, "Charge the battery of a robot".

If anyone develops a bull robot for a rodeo they might want "charge" to mean something else.